frontend/src/pages/accounts/details/Details.tsx

```diff
@@ -210,11 +228,35 @@ const Details: FunctionComponent = () => {
           </Fragment>
         </DetailsTabItem>
         <DetailsTabItem label="STORAGE" value={account.storage?.length}>
-          <Table<DecoratedPollingEntity<AccountStorageItem>>
-            className={classes.storageTable}
-            data={storageItems}
-            columns={columnsStorage}
-          />
+          <div className={classes.gridStorage}>
+            {storageItems &&
+              storageItems.map((item) =>
+                FlowUtils.getLowerCasedPathDomain(item.pathDomain) ==
+                  "private" ||
```

Comment:
This comparison is not type-safe. In case "path domain" type definition changes, TypeScript won't warn you about this. It's better to compare the enum types directlty, like so:

```ts
item.pathDomain == AccountStorageDomain.STORAGE_DOMAIN_STORAGE
```

Also here, you are basicly filtering the list of all storage items, to show just the ones you are interested in.
I think it's better to compute this upfront, so that it's more readable, can be reused in other logic (if needed) and because that allows you to enforce a certain ordering of items.

You could rewritten this code, as follows:
```jsx
// declared somwehere at the beginning of the component
const privateStorageItems = storageItems.map(item => item.pathDomain === AccountStorageDomain.STORAGE_DOMAIN_PRIVATE);
const publicStorageItems = storageItems.map(item => item.pathDomain === AccountStorageDomain.STORAGE_DOMAIN_PUBLIC);
const storageItems = storageItems.map(item => item.pathDomain === AccountStorageDomain.STORAGE_DOMAIN_STORAGE);

// Note that we can enforce the ordering of items here
// Public ones will appear before the private ones, which could be usefull from the user's perspective
const privateOrPublicStorageItems = [...publicStorageItems, ...privateStorageItems];

// ... other code ...

return (
 // use the computed state in the actuall HTML
)
```

Does that make sense?